modules/cluster-logging-collector-alerts.adoc

```diff
@@ -8,29 +8,29 @@
 
 The following alerts are generated by the logging collector. You can view these alerts in the {product-title} web console, on the *Alerts* page of the Alerting UI.
 
-.Fluentd Prometheus alerts
+.collector Prometheus alerts
 [cols="2,2,2,1",options="header"]
 |===
 |Alert |Message |Description |Severity
 
-|`FluentDHighErrorRate`
-|`<value> of records have resulted in an error by fluentd <instance>.`
-|The number of FluentD output errors is high, by default more than 10 in the previous 15 minutes.
+|`collectorHighErrorRate`
```

Comment:
The alert names are still Fluent-xxxx. Filed a bug https://issues.redhat.com/browse/LOG-3127